### ISSUE TITLE
[10.0] ImportError: No module named QWebTemplateNotFound

### ISSUE THREAD

**Issue by Yenthe666** [odoo/odoo#13701]
Impacted versions: 10.0

Steps to reproduce:
Install Odoo V10 Enterprise and try to launch it

Current behavior:
An internal server error pops up because Odoo cannot find QWebTemplateNotFound:
![image](https://cloud.githubusercontent.com/assets/6352350/19187666/17d8a3e2-8c8d-11e6-9696-447ba760142d.png)

Expected behavior:
Odoo launches.

Stacktrace:
```
  File "/odoo-ent/odoo-ent-server/odoo/modules/module.py", line 109, in load_module
    mod = importlib.import_module(canonical)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/odoo-ent/enterprise/addons/web/controllers/__init__.py", line 1, in <module>
    from . import main, pivot
  File "/odoo-ent/enterprise/addons/web/controllers/main.py", line 30, in <module>
    from openerp.addons.base.ir.ir_qweb import AssetsBundle, QWebTemplateNotFound
  File "/odoo-ent/odoo-ent-server/odoo/modules/module.py", line 109, in load_module
    mod = importlib.import_module(canonical)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
```

See https://github.com/odoo/odoo/search?utf8=%E2%9C%93&q=QWebTemplateNotFound

**Comment by Yenthe666** [comment]
@mart-e @JKE-be and @odony quite a big one, breaks all new installations.
I don't see the issue right away.

**Comment by Yenthe666** [comment]
Ignore this, I made a mistake. My apoligies. 😄

**Comment by atchuthan** [comment]
I too had the same problem for 10.0 enterprise.

The problem was due to odoo code checkout to 10.0(using old github code in 9.0), thus creating empty modules e.g. web and other modules with only *.pyc files.

Reset the code available in 10.0-enterprise to remove empty modules with only *.pyc

Hope, this helps others too.
